.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,6 +8,13 @@ assignees: ''
 
 **Describe the bug** A clear and concise description of what the bug is.
 
+**Version number** The version of Lightning you have seen the bug on.
+
```

Comment:
Add two examples: 
"""
If you're running on a tagged version, include the number: `X.Y.Z`
If you're running locally with an unreleased version, the commit sha: `84b1bdc`